app/Commands/StopCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Commands;
+
+use App\InitializesCommands;
+use App\Services;
+use App\Shell\Docker;
+use Illuminate\Support\Str;
+use LaravelZero\Framework\Commands\Command;
+use PhpSchool\CliMenu\CliMenu;
+
+class StopCommand extends Command
```

Comment:
Why do we want people to be able to stop services?

Reply:
Additionally, if this were to stay, you'd also need to add it to the notes in the help command.

Reply:
If I may, my use case for `stop`:

- I have an old MySQL container only needed when I work on legacy apps.
- A stopped container doesn't consume RAM nor CPU so I can keep it around for a while.
- It's easier to start a stopped container than to spin-up a new one and having to go through port, version & volume configuration again.

Reply:
I agree with @AlexMartinFR. My personal laptop is a 2011 MacBook and if you mistype the volume name when reenabling the service would cause the data to not be loaded as expected.

Reply:
This is a pretty compelling case. Thanks!